starter_code/src/movies.js

```diff
@@ -1,20 +1,135 @@
 /* eslint no-restricted-globals: 'off' */
-// Turn duration of the movies from hours to minutes 
 
+// -------------------------------------------- //
+// Turn duration of the movies from hours to minutes  
+// -------------------------------------------- // 
+const setDurationInMinutes = arrayInput => (
+  arrayInput.map(element => {
+    let hour = parseInt(element.duration.slice(0, element.duration.indexOf("h")) * 60) || 0;
```

Comment:
Your code seems fine, but why did you use the (OR) '||' operator at the end of this line? As far as I can see this is not needed. 

What you could have done as error handling is to use a ternary operator that returns 0 or any other value in case of an 'error' or 'undefined'.

Reply:
Not every movie contained hours, some only contained minutes. 
If the parseInt(...) returns undefined (false) because "h" was not found it should return 0. 

In my testing this seemed to work. For me it's seems rather similar to the ternary operator